beluga/include/beluga/algorithm/spatial_hash.hpp

```diff
@@ -196,6 +196,61 @@ class spatial_hash<Sophus::SE2d, void> {
   spatial_hash<std::tuple<double, double, double>> underlying_hasher_{{1., 1., 1.}};
 };
 
+/**
+ * Specialization for Sophus::SE3d. Will calculate the spatial hash based on the translation and rotation (expressed in
```

Comment:
@serraramiro1 nit 
```suggestion
 * Specialization for Sophus::SE3d. It will calculate the spatial hash based on translation and rotation (expressed in
```